package.json

```diff
@@ -62,6 +63,7 @@
     "express-session": "^1.17.2",
     "express-slow-down": "^1.4.0",
     "express-validator": "^6.12.0",
+    "faker": "^5.5.3",
```

Comment:
Shouldn't these be in dev dependencies?

Reply:
not really, the dev dependencies are used for pre release stuff. The way our code is done allow for the npm run seed to be run anytime